### PR TITLE
fix: change uaxl to uamplifier in verifier onboarding

### PR DESCRIPTION
**Pull request by benjamin852**

### CODE REVIEW

File: src/content/docs/validator/amplifier/verifier-onboarding.mdx [hunk 1]
```diff
@@ -450,17 +450,27 @@ Prior to running the `ampd` daemon, you will need to set up your wallet with dev
...
 
-   ```bash
-   ampd bond-verifier amplifier 100000000000 uaxl
-   ```
-
-   Bonded verifiers will stay bonded until they unbond or are removed by governance. The bond for testnet is 100000000000uaxl (100k AXL). The bond for mainnet is 50000000000uaxl (50k AXL).
+   <tabs>
+   <tab-item title="devnet-amplifier">
+      ```bash
+      ampd bond-verifier validators 100000000000 uamplifier
```
Comment (StephenFluin) [inline src/content/docs/validator/amplifier/verifier-onboarding.mdx:456]: isn't the testnet bond 100 uamplifier?

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: src/content/docs/validator/amplifier/verifier-onboarding.mdx (hunk 1 above)
Comment (benjamin852) [reply src/content/docs/validator/amplifier/verifier-onboarding.mdx:456]: double checked it's `uaxl` for testnet `uamplifier` is only for devnet

**Review by canhtrinh — APPROVED** [review]

**Comment by canhtrinh** [comment]
> @benjamin852 I disagree with this change. The instructions here are for the verifiers, which should be onboarding to testnet and then mainnet, so the default instructions are for testnet. Verifiers shouldn't onboard to a devnet. I think we can copy / link from the integrator docs with caveats if we need something for the devnet, or create a dedicated section/tab for it.

i think the change makes sense, at least as an interim update that clarifies that params are different for devnets. there is way too much noise in the integrators Slack channel of integrators trying to add a verifier on devnet-amplifier with testnet values. can follow up with your idea of a dedicated section specifically for integrators later

**Comment by StephenFluin** [comment]
> > @benjamin852 I disagree with this change. The instructions here are for the verifiers, which should be onboarding to testnet and then mainnet, so the default instructions are for testnet. Verifiers shouldn't onboard to a devnet. I think we can copy / link from the integrator docs with caveats if we need something for the devnet, or create a dedicated section/tab for it.
> 
> i think the change makes sense, at least as an interim update that clarifies that params are different for devnets. there is way too much noise in the integrators Slack channel of integrators trying to add a verifier on devnet-amplifier with testnet values. can follow up with your idea of a dedicated section specifically for integrators later

@benjamin852 added tabs which 100% addressed my concerns.

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**